### DISCUSSION TITLE
What is the duration of Play 2.x line maintenance?

### DISCUSSION THREAD

**Discussion by szielinski** [playframework/playframework#12912] [Q&A]
Hi, the document at https://www.playframework.com/documentation/3.0.x/General#End-of-life-(EOL)-Dates mentions that "we will evaluate the duration of Play 2.x line maintenance in September 2024", has that discussion happened yet and what was the outcome? Thank you

**Comment by mkurz** [comment]
It looks like we'll continue with the 2.x line also with next Play major release just like we do right now.

**Reply by szielinski** [reply]
Thank you
